packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -245,12 +244,7 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
                 })
               ) : (
                 <div className="mr-2">
-                  <Loader
-                    type="Oval"
-                    color="rgb(40, 160, 240)"
-                    height={14}
-                    width={14}
-                  />
+                  <Loader color="#28A0F0" size="small" />
```

Comment:
are we sure we want to change it from 16px to 14px?
if we need to keep 14px, we can add xsmall = 14px

Reply:
A 2px size change wouldn't make a lot of difference here, visually. When (and if) the need arises, I'd prefer `xsmall` to have a good size difference compared to `small`.